From: Andreas Gruenbacher <agruen@suse.de>
Subject: Remove an unused wait queue variable

The "real" wait queue is defined some lines further down inside the loop.

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

Index: linux-2.6.11-latest/fs/mbcache.c
===================================================================
--- linux-2.6.11-latest.orig/fs/mbcache.c
+++ linux-2.6.11-latest/fs/mbcache.c
@@ -554,8 +554,6 @@ static struct mb_cache_entry *
 __mb_cache_entry_find(struct list_head *l, struct list_head *head,
 		      int index, struct block_device *bdev, unsigned int key)
 {
-	DEFINE_WAIT(wait);
-
 	while (l != head) {
 		struct mb_cache_entry *ce =
 			list_entry(l, struct mb_cache_entry,
